src/kixi/heimdall/group.clj

```diff
@@ -15,6 +14,14 @@
     (db/insert! session :groups group-data)
     {:group-id group-id}))
 
+(defn update!
+  [session group-id {:keys [name]}]
+  (db/update! session :groups {:name name} {:id (java.util.UUID/fromString group-id)}))
+
 (defn find-by-id
   [session id]
   (first (db/select* session :groups {:id id}))  )
+
+(defn all
```

Comment:
no, it's for the groups end point (as in the search).
It should change to 'everything you can see' as soon as I've done permissions ...